tests/e2e-integration/e2e/sections/create-dataset/CreateDatasetForm.spec.tsx

```diff
@@ -19,11 +19,29 @@ describe('Create Dataset', () => {
   it('navigates to the new dataset after submitting a valid form', () => {
     cy.visit('/spa/datasets/create')
 
-    cy.findByLabelText(/Title/i).type('Test Dataset Title')
-    cy.findByLabelText(/Author Name/i).type('Test author name', { force: true })
-    cy.findByLabelText(/Point of Contact E-mail/i).type('email@test.com')
-    cy.findByLabelText(/Description Text/i).type('Test description text')
-    cy.findByLabelText(/Arts and Humanities/i).check()
+    cy.findByLabelText(/^Title/i).type('Test Dataset Title', { force: true })
+    cy.findByText('Author')
+      .closest('.row')
+      .within(() => {
```

Comment:
Consider using floating labels to look for the inputs by their labels

Reply:
This is only for composed fields, where for example in this case, 'Author' is the root 'label' lets say but is not actually a label.

Reply:
Yes, I understand. That's why I was suggesting the use of [Floating Labels](https://react-bootstrap.netlify.app/docs/forms/floating-labels) because each input needs to have its own label. "Author" cannot be the label for four different inputs.

We can use floating labels to provide each input with its own label "Author name", "Author Affiliation", without significantly altering the UI, as these labels are floating.

Reply:
Oh, I understand now what you mean, labels from composed fields groups may not be unique. Let discuss it with the team, great!

Reply:
@GermanSaracca  I think that change doesn't need to block this PR. I'll approve the PR, and you can discuss it with the team. If there is an action item, you can create an issue. Thanks!